index.html

```diff
@@ -0,0 +1,194 @@
+             <button class="btn"><div class="senta">See Project<div></button>
+             
+        </div>
+        </section>
+
+        <section class="about-me">
+        <div class="about-me-container">
+            <section class="about-1">
+                 <h1 class="about-header">About Myself</h1>
+                 <p class="about-description"> Hello I'm a software devoloper! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+                 <p class="connect-2">LET'S CONNECT</p>
+                 <ul class="connect-icons">
+                    <li><a href="https://twitter.com/JosiahMutinhiri"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/josiah-mutinhiri-536387153/"><i class="fa-brands fa-linkedin"></i></a></li>
+                    <li><a href="https://github.com/josman227"><i class="fa-sharp fa-solid fa-m"></i></i></a></li>
+                    <li><a href="https://www.facebook.com/josiah.mutinhiri"><i class="fa-brands fa-facebook"></i></a></li>
+                    <li><a href="https://github.com/josman227"><i class="fa-brands fa-github"></i></a></li>
+                    <li><a href="https://github.com/josman227"><i class="fa-brands fa-angellist"></i></a></li>
+                 </ul>
+                 <button class=" btn resume-btn">Get my resume</button>
+            </section>
+            <section class="about-2">
+             <ul class="tools">
+                     <div class="lang-arrow">
+                         <h3>Languages</h3>
+                         <img src="img/arrow.png" class="ira" alt="">
+                     </div>
+                     
+                     <ul class=langua>
+                         <li class="lang-item">
+                             <img class="ipin" src="img/javascript-logo.png" alt="lang">
+                             <p>JavaScript</p>
+                         </li>
+                         <li class="lang-item">
+                             <img class="ipin" src="img/html-logo.png" alt="lang">
+                             <p>HTML</p>
+                         </li>
+                         <li class="lang-item">
+                             <img class="ipin" src="img/css-logo.png" alt="lang"> 
+                             <p>CSS</p> 
+                         </li>
+                     </ul>
+                 </li>
+                 <li>
+                     <div class="lang-point">
+                         <h3>Frameworks</h3>
+                           <img src="img/Separator.png" alt="">
+                     </div>
+                     <hr>
+                 </li>
+                 <li>
+                     <div class="lang-point">
+                         <h3>Skills</h3>
+                         <img src="img/rightArrow.png" alt="">
+                     </div>
```

Comment:
Figma
![image](https://user-images.githubusercontent.com/78251927/229295378-6a67166c-8ba0-4710-8940-f1b110ed3e32.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/229295437-721e7389-6459-4887-8ad7-7cde4bcee337.png)

As requested in the previous review, please kindly place the highlighted icon in the correct position using the Figma design and add the missing one. (The `icon` at the front of the `Frameworks` text). 🤗